mantidproject/fitbenchmarking
# Removed unused problems

#### Description of Work:
Fixes #35
Removed the unused problems from the code and got rid of the extra code that dealt with special cases brought about in these problems. Additionally, improved the parsing nist data function

#### Testing Instructions:
1. In `example_runScript.py`, make sure that `run_data = "nist"`.
2. In the mantidpython console, run `example_runScript.py`.
3. Check if the resulting tables are identical with the ones produced by running master with nist data.
